Dockerfile

```diff
@@ -53,6 +53,21 @@ ENTRYPOINT ["poetry", "run", "pytest", "tests/unit", "--junit-xml", "test_result
 # RUN TESTS
 FROM base as integrationtests
 
+ENV FLASK_ENV=development \
+    PYTHONPATH=/app
+
+COPY todoapp /app/todoapp
```

Comment:
As with your dev/prod, it's usually best to delay this step until as late as possible (in particular after the poetry install) so that changing your code doesn't mean your next docker build has to reinstall dependencies (which is typically slow)